astroquery/conftest.py

```diff
@@ -20,25 +19,6 @@
 except NameError:
     pass
 
-
-# pytest magic:
-# http://pytest.org/latest/plugins.html#_pytest.hookspec.pytest_configure
-# use pytest.set_trace() to interactively inspect config's features
-def pytest_configure(config):
-    if config.getoption('remote_data'):
-        pass
-    else:
-        turn_off_internet(verbose=config.option.verbose)
```

Comment:
As far as I understand this is taken care of in ``astropy.tests.pytest_plugins`` (implemented by you in https://github.com/astropy/astropy/pull/2044 :) ) . E.g. https://github.com/astropy/astropy/blob/master/astropy/tests/pytest_plugins.py#L140

```
    if not config.getoption('remote_data'):
        turn_off_internet(verbose=config.option.verbose)
```